u6-spas/src/views/Converter.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <table>
+    <tr>
+      <td><input v-model="input" /></td>
```

Comment:
คุณควรใช้ `v-model.number` ไม่เช่นนั้นค่าที่ผูกไว้จะเป็น `string`
You should use `v-model.number`, otherwise the bound value will be a `string`.